src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -407,5 +407,23 @@ public override IReadOnlyList<object> ValidatorMetadata
                 return _validatorMetadata;
             }
         }
+
+        /// <inheritdoc />
+        public override Func<object, object> PropertyAccessor
+        {
+            get
+            {
+                return _details.PropertyAccessor;
+            }
+        }
+
+        /// <inheritdoc />
+        public override Action<object, object> PropertySetter
+        {
+            get
+            {
+                return _details.PropertySetter;
+            }
```

Comment:
Would like to see some unit tests for when these are populated and not. Take a look at `ModelMetadataProviderTest` which has more of the integration type tests.